app/javascript/packs/document-capture-welcome.ts

```diff
@@ -33,21 +33,15 @@ function addFormInputsForMobileDeviceCapabilities() {
   // submit() to ensure that it completes in time.
   const cameraCheckPromise = measure(hasCamera).then(
```

Comment:
Do we care about this check anymore? Otherwise, kinda feels like this file could generally boil down to...

```ts
import { isCameraCapableMobile } from '@18f/identity-device';

if (isCameraCapableMobile()) {
  const input = document.createElement('input');
  input.type = 'hidden';
  input.name = 'skip_upload';
  form.appendChild(input);
}
```

Reply:
Yes, that would work.  Looks like over the last 4 weeks, 99.97% of these checks where we called the device "camera capable mobile" also then subsequently reported actually having a camera present. My vote would be to replace this file with what @aduth wrote above (including scrapping the analytics call)

Reply:
Chiming in to note: my dashboards that monitor the hybrid handoff don't use this analytics call. So, go ahead and delete it.

Reply:
I made a commit to accept @aduth's suggestion on more code deletion above, and then realized it's out of scope for this ticket. I made a new ticket [LG-9567](https://cm-jira.usa.gov/browse/LG-9567), and I'll force push to remove that commit before merging. Thanks everyone for taking a look.